AC_PROG_CC will initialise a lot of value which is used by
- AC_HEADER_STDC; and
- AC_CHECK_SIZEOF
---
--- a/configure.in
+++ b/configure.in
@@ -5,9 +5,7 @@ cp $srcdir/configure.sub $srcdir/config.sub
 
 AC_CANONICAL_HOST
 
-if test -z "$CC"; then
-	AC_PROG_CC	
-fi
+AC_PROG_CC
 AC_PROG_RANLIB
 AC_CHECK_PROG(AR,ar,ar)
 AC_CHECK_PROG(INSTALL,install,install)
